client/src/main/resources/server/server-compose.yml

```diff
@@ -0,0 +1,8 @@
+name: kurtama-server
+services:
+  kurtama-server:
+    image: ghcr.io/schlunzis/kurtama/server:v{version}
+    container_name: kurtama-server
+    ports:
+      - '{port}:8007'
+#    volumes:
```

Comment:
```suggestion
```
As we don't want to have persistent storage for the server, we don't need to have volumes, especially not as a comment.